### ISSUE TITLE
no page is shown when login fails

### ISSUE THREAD

**Issue by asana17** [supersaintex/saint-sorting#1]
When login fails, no page is shown. This is because the user is accessing /app/top/login page after login, and the user does not have accessibility.

**Comment by asana17** [comment]
This issue is resolved at https://github.com/supersaintex/saint-sorting/pull/15.
